docs/self-managed/platform-deployment/openshift.md

```diff
@@ -0,0 +1,73 @@
+
+## Permissive SCC
+
+Out of the box, if you will deploy Camunda 8 (and related infrastructure) with a permissive SCC, then there is nothing special for you to configure. Here, a permissive SCC refers to one where the strategy for `RunAsUser` is defined as `RunAsAny` (including root).
+
+## Non-root SCC
+
+If you wish to deploy Camunda 8 but restrict the applications from running as root (e.g. the `nonroot` built-in SCC), then you will need to configure each pod and container to run as a non-root user. For example, when deploying Zeebe using a stateful set, you would add the following, replacing `1000` with the user ID you wish to use:
+
+```yaml
+spec:
+  template:
+    spec:
+      securityContext:
+        runAsUser: 1000
+      containers:
+        securityContext:
+          runAsUser: 1000
+```
+
+:::note
+As the container user in OpenShift is always part of the root group, it's not necessary to define an `fsGroup` for any Camunda 8 applications' pod security context.
+:::
+
+This is necessary for all Camunda 8 applications, as well as related ones (e.g. Keycloak, Postgresql, etc.). This is notably crucial for stateful applications which will write to persistent volumes, but it's also generally a good idea to set.
+
+## Restrictive SCC
+
+The following is the most restrictive SCC you can use to deploy Camunda 8. Note that this is, in OpenShift 4.10, equivalent to the built-in `restricted` SCC (which is the default SCC).
+
+```yaml
+Allow Privileged: false
+Default Add Capabilities: <none>
+Required Drop Capabilities: KILL,MKNOD,SYS_CHROOT,SETUID,SETGID
+Allowed Capabilities: <none>
+Allowed Seccomp Profiles: <none>
+Allowed Volume Types: configMap,downwardAPI,emptyDir,persistentVolumeClaim,projected,secret
+Allow Host Network: false
+Allow Host Ports: false
+Allow Host PID: false
+Allow Host IPC: false
+Read Only Root Filesystem: false
+Run As User Strategy: MustRunAsRange
+SELinux Context Strategy: MustRunAs
+FSGroup Strategy: MustRunAs
+Supplemental Groups Strategy: RunAsAny
+```
+
+When using this, you must take care not to specify _any_ `runAsUser` or `fsGroup` in either the pod or container security context. Instead, let OpenShift assign arbitrary IDs.
+
+:::note
+Of course, if you are providing the ID ranges yourself, you can configure the `runAsUser` and `fsGroup` values yourself as well.
+:::
+
+If you are using Helm charts to deploy non-Camunda applications (e.g. Keycloak, Elasticsearch, etc.), refer to their own Helm charts on how to configure it for OpenShift. However, note that the recommendations listed above also apply, and you will also need to ensure that `runAsUser` and `fsGroup` are appropriately configured.
```

Comment:
It's hard to understand this section.

Reply:
Can you elaborate? Is it the phrasing, or should I define some nomenclature, or is it throwing in concepts I haven't explained yet, etc.?

Reply:
The sentence is hard to read/understand :sweat_smile: 

> If you are using Helm charts to deploy non-Camunda applications (e.g. Keycloak, Elasticsearch, etc.),

Do you mean within Camunda Platform charts or if someone installed these components externally? (outside C8SM charts) 

It's not clear because we actually use external charts for non-Camunda apps. (e.g. Keycloak, Elasticsearch, etc.)

> refer to their own Helm charts on how to configure it for OpenShift.

The word `refer` here means `follow`? If yes, then I'd suggest to use `follow`
because when I read `refer` I think about editing the C8SM chart dependencies and replace the external charts there (e.g. if someone want to use homemade ES chart instead of the official one).

Reply:
I'm thinking of just dropping this block. I'm not sure it's all that useful, actually: should we document how to deploy Elasticsearch standalone on OpenShift, _without our Helm charts_? I'm just not sure it's valuable. So I'd opt to simply remove it.

Reply:
I agree to remove it :+1: